src/parsers/js/transformers/babel6/index.js

```diff
@@ -26,7 +26,7 @@ export default {
     );
 
     return babel.transform(code, {
-      presets,
+      presets: [],
```

Comment:
I think we'll need to modify `halting-problem` to allow using `babylon` instead like in https://github.com/ForbesLindesay/halting-problem/blob/master/lib/halting-problem.js#L3? cc @ForbesLindesay